tools/hangar/__snapshots__/test_corpus/valid/struct_from_json.w_compile_tf-aws.md

```diff
@@ -629,10 +541,17 @@ const $plugins = ((s) => !s ? [] : s.split(';'))(process.env.WING_PLUGIN_PATHS);
 const $outdir = process.env.WING_SYNTH_DIR ?? ".";
 const $wing_is_test = process.env.WING_IS_TEST === "true";
 const std = $stdlib.std;
+const cloud = $stdlib.cloud;
 const externalStructs = require("./preflight.structs-1.js")({ $stdlib });
 class $Root extends $stdlib.std.Resource {
   constructor(scope, id) {
     super(scope, id);
+    const Foo = require("./Foo.Struct.js")($stdlib.std.Struct);
+    const Bar = require("./Bar.Struct.js")($stdlib.std.Struct);
+    const BucketProps = require("./BucketProps.Struct.js")($stdlib.std.Struct);
+    const Student = require("./Student.Struct.js")($stdlib.std.Struct);
+    const MyOtherStruct = require("./MyOtherStruct.Struct.js")($stdlib.std.Struct);
+    const Foosible = require("./Foosible.Struct.js")($stdlib.std.Struct);
```

Comment:
The order that these are emitted does not seem deterministic (see all the mutations on this PR)

Reply:
I didn't check the code yet, but make sure any structures you're iterating over to jsify use a data structure with deterministic iteration (like BTreeMap over HashMap)